src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -236,27 +247,9 @@ instance MetaOrObject level => PrettyPrint (Variable level) where
             , writeFieldNewLine "variableSort" variableSort var
             ]
 
-instance PrettyPrint UnifiedSortVariable where
-    prettyPrint flags (ObjectSortVariable sv) =
-        writeOneFieldStruct flags "ObjectSortVariable" sv
-    prettyPrint flags (MetaSortVariable sv)   =
-        writeOneFieldStruct flags "MetaSortVariable" sv
-
-instance PrettyPrint (UnifiedVariable Variable) where
-    prettyPrint flags (ObjectVariable sv) =
-        writeOneFieldStruct flags "ObjectVariable" sv
-    prettyPrint flags (MetaVariable sv)   =
-        writeOneFieldStruct flags "MetaVariable" sv
-
-instance PrettyPrint UnifiedPattern where
-    prettyPrint flags (ObjectPattern sv) =
-        writeOneFieldStruct flags "ObjectPattern" sv
-    prettyPrint flags (MetaPattern sv)   =
-        writeOneFieldStruct flags "MetaPattern" sv
-
 instance
-    ( MetaOrObject level
-    , PrettyPrint child
+    ( PrettyPrint child
```

Comment:
Just curious: any reason to change their order?

Reply:
Not really :-(  I was trying a different approach and removed MetaOrObject constraint.  When that did not work, I added back, and it was simpler to add it this way (no need to change again existing lines.  If you want, I can try making it as it was originally.